src/mongocrypt-status.c

```diff
@@ -148,3 +148,17 @@ mongocrypt_status_destroy (mongocrypt_status_t *status)
    bson_free (status->message);
    bson_free (status);
 }
+
+
+void
+_mongocrypt_status_wrap (mongocrypt_status_t *status,
```

Comment:
So, at the end of this method, `to_wrap` is unchanged, and `status` contains a message like `status's original message: to_wraps' message`

Could you add a comment explaining this?  It's a little difficult to parse reading it.  Maybe changing the names of the parameters could help, too?  What do you think of calling this "append" instead of "wrap" ?

Reply:
Good idea. "append" seems much clearer. Updated and added a function comment.